docs/source/alfacase/case_description.example.rst

```diff
@@ -3,5 +3,439 @@
+
+    In this case, the application will look for the ``.tab`` file at  ``C:\Users\alfasim\my_pvt_file.tab``
+
+    .. code-block:: yaml
+
+        tables:
+           # PVT name  : file path
+          'Pvt1': my_pvt_file.tab
+
+
+.. note::
+
+    The PVT name must be unique.
+
+    Check the :ref:`alfacase-reference-pvt-section` section for a detailed description of each PVT type option.
+
+
+.. rubric:: Example
+
+
+.. code-block:: python
+
+    from alfasim_sdk import (
+        CaseDescription,
+        PvtModelsDescription,
+    )
+
+    CaseDescription(
+        # Omitting fields that configure option
+        pvt_models=PvtModelsDescription(
+            default_model="Pvt1",
+            tables={"Pvt1": "my_pvt_file.tab"},
+        ),
+    )
+
+
+Nodes
+-----
+
+The third step will add two different types of nodes, a mass source node, and a pressure node.
+
+All nodes that will be used on the application need to be added over the ``nodes`` section of the
+:class:`CaseDescription <alfasim_sdk.CaseDescription>`.
+
+The :class:`NodeDescription <alfasim_sdk.NodeDescription>` is responsible to configure several types of nodes
+through the ``node_type`` field and their respective property fields.
+
+For example, when the ``node_type`` is  ``NodeCellType.MassSource``, besides the fields from :class:`NodeDescription <alfasim_sdk.NodeDescription>`
+only the fields available at ``mass_source_properties`` will be considered.
+
+And when ``node_type`` is ``NodeCellType.Pressure`` only the fields from  ``pressure_properties`` will be considered.
+
+
+.. note::
+
+    Check the :ref:`alfacase-reference-node-section` section for a detailed description of each ``Node`` type.
+
+.. rubric:: Example
+
+.. code-block:: python
+
+    from alfasim_sdk import (
+        CaseDescription,
+        NodeDescription,
+        MassSourceNodePropertiesDescription,
+        PressureNodePropertiesDescription,
+    )
+
+    CaseDescription(
+        # Omitting fields that configure Options and Pvt
+        nodes=[
+            NodeDescription(
+                name="Inlet",
+                node_type=NodeCellType.MassSource,
+                mass_source_properties=MassSourceNodePropertiesDescription(
+                    mass_flow_rates={
+                        "gas": Scalar(0.0, "kg/s"),
+                        "oil": Scalar(0.0, "kg/s"),
+                    },
+                ),
+            ),
+            NodeDescription(
+                name="Outlet",
+                node_type=NodeCellType.Pressure,
+                pressure_properties=PressureNodePropertiesDescription(
+                    pressure=Scalar(50.0, "bar"),
+                    volume_fractions={
+                        "gas": Scalar(1.0, "-"),
+                        "oil": Scalar(1.0, "-"),
+                    },
+                ),
+            ),
+        ],
+    )
+
+
+Pipes
+-----
+
+The fourth step will add a Pipe to the application through the ``pipes`` fields.
```

Comment:
We could remove references to this "steps order" in case we add something in between later.

```suggestion
Let's add a Pipe to the application through the ``pipes`` fields.
```